src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-size: normal;
+  font-weight: bold;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+form {
+  margin: 20px 0;
+}
```

Comment:
use class as selector here instead of tag

Reply:
Sorry, i put the class in HTML but forgot to put point in CSS